### ISSUE TITLE
raise RuntimeError('Error(s) in loading state_dict for {}:\n\t{}'.format

### ISSUE THREAD

**Issue by manhnguyen92** [zengxianyu/crfill#29]
raise RuntimeError('Error(s) in loading state_dict for {}:\n\t{}'.format(
RuntimeError: Error(s) in loading state_dict for BaseConvGenerator:
	size mismatch for conv14.weight: copying a param with shape torch.Size([96, 96, 3, 3]) from checkpoint, the shape in current model is torch.Size([96, 48, 3, 3]).
	size mismatch for conv16.weight: copying a param with shape torch.Size([48, 48, 3, 3]) from checkpoint, the shape in current model is torch.Size([24, 24, 3, 3]).
	size mismatch for conv16.bias: copying a param with shape torch.Size([48]) from checkpoint, the shape in current model is torch.Size([24]).
	size mismatch for conv17.weight: copying a param with shape torch.Size([3, 24, 3, 3]) from checkpoint, the shape in current model is torch.Size([3, 12, 3, 3]).

**Comment by 1997Jessie** [comment]
I have same problems with you! Hava u solved it?

**Comment by manhnguyen92** [comment]
@1997Jessie  yes i did. but result image not good as expected. you should find another lib

**Comment by 1997Jessie** [comment]
> @1997Jessie yes i did. but result image not good as expected. you should find another lib

copy that！
